.github/docker/docker-compose.yml

```diff
@@ -0,0 +1,67 @@
+version: '3.8'
+
+services:
+  db:
+    container_name: db
+    ports:
+      - 3306:3306
+    image: mysql:5.7
+    environment:
+      MYSQL_ROOT_PASSWORD: ${WORDPRESS_DB_ROOT_PASSWORD}
+      MYSQL_DATABASE: ${WORDPRESS_DB_NAME}
+      MYSQL_USER: ${WORDPRESS_DB_USER}
+      MYSQL_PASSWORD: ${WORDPRESS_DB_PASSWORD}
+    volumes:
+      - db_data:/var/lib/mysql
+
+  wordpress:
+    depends_on:
+      - db
+    container_name: wordpress
+    image: wordpress:latest
+    environment:
+      WORDPRESS_DB_HOST: ${WORDPRESS_DB_HOST}
+      WORDPRESS_DB_NAME: ${WORDPRESS_DB_NAME}
+      WORDPRESS_DB_USER: ${WORDPRESS_DB_USER}
+      WORDPRESS_DB_PASSWORD: ${WORDPRESS_DB_PASSWORD}
+      WORDPRESS_DEBUG: ${WORDPRESS_DEBUG}
+      WORDPRESS_CONFIG_EXTRA: |
+        define('WP_HOME', 'http://${WEB_HOST}');
+        define('WP_SITEURL', 'http://${WEB_HOST}');
+        define('FORCE_SSL_ADMIN', false);
+    volumes:
+      - wp_data:/var/www/html
+
+  wordpress-cli:
```

Comment:
In case anyone asks, the reason why I have the `wordpress-cli` in another container instead of building a custom dockerfile with `wordpress` on it. It's because it's currently not possible to have `wordpress-cli` run the setup when building the image because `wordpress-cli` also needs to connect to the database to store data.